app/models/slot.rb

```diff
@@ -9,11 +9,20 @@ def self.all
   end
 
   def self.slot_dates
-    grace_period = Time.zone.today + 2.days
     booking_window_end = 6.weeks.from_now
 
     (grace_period..booking_window_end).reject do |date|
       date.saturday? || date.sunday?
     end
   end
+
+  def self.grace_period
```

Comment:
Do we need to cater for Thursday -> Tuesday? I think it's 3 working days they're after rather than just 3 days. Pretty sure this is enough for now though as bank holidays will also become a consideration. @mattlucht any thoughts?